sass/_content.scss

```diff
@@ -0,0 +1,68 @@
+.content {
+    
```

Comment:
Avoid adding empty lines inside on element. Finally it will be compiled in a minified file of course, so there is no impact on the product. But it is difficult to read such code.
Suggestion:
```
.content {
    padding: 2.5rem;
    display: grid;
    grid-template-columns: .23fr .44fr .33fr; 
    grid-gap: 1.1rem;
    grid-auto-flow: dense;
    
    @media(max-width: 992px) { ...
```